src/main/webapp/js/CryptoJS/rollups/md5.js

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Do we need to add these obfuscated js codes into our repo or they can be loaded dynamically?